modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/macros/Masterclass.macro

```diff
@@ -25,6 +25,54 @@ definition {
 			siteTitle = "${siteName}");
 	}
 
+	macro createAccount {
```

Comment:
don't we have a Jason method for this?

Reply:
But the testcase is to create the account by UI, right?